### DISCUSSION TITLE
Remove Actionable  Notifications Screen on Fire Cube

### DISCUSSION THREAD

**Discussion by Krispkiwi** [keatontaylor/alexa-actions#185] [Q&A]
Whenever you load a actionable notification on Firecube with the TV on, it brings up a screen with a grey background with "alexa actionable notifications in text on it. Is it possible to prevent this page from loading up so it stays on the screen it was already on but stlil plays the script?

**Comment by DEADSEC-SECURITY** [comment]
This would have to be  modified in the skill code itself, currently we do not support this feature and we are not exactly sure how to achieve this yet, but if you have done any research or anyone who might know how to help we would really appreciate your comment/PR to the project.
